reference/docs-conceptual/getting-started/fundamental/Getting-Information-About-Commands.md

```diff
@@ -40,21 +50,35 @@ Get-Help [[-Name] <String>] [-Path <String>] [-Category <String[]>] [-Component
  [-Role <String[]>] [-Parameter <String>] [-Online] [-Verbose] [-Debug] [-ErrorAction <ActionPreference>] [-WarningAction <ActionPreference>] [-ErrorVariable <String>] [-WarningVariable <String>] [-OutVariable <String>] [-OutBuffer <Int32>]
 ```
 
-### Displaying Available Command Types
-The **Get-Command** command does not list every command that is available in Windows PowerShell. Instead, the **Get-Command** command lists only the cmdlets in the current session. Windows PowerShell actually supports several other types of commands. Aliases, functions, and scripts are also Windows PowerShell commands, although they are not discussed in detail in the Windows PowerShell User's Guide. External files that are executable, or have a registered file type handler, are also classified as commands.
+## Displaying available command by type
+
+The **Get-Command** command lists only the cmdlets in the current session. PowerShell actually
```

Comment:
Get-Command should be code-fenced and not bold like a property.

Reply:
done